operators/advanced_network/advanced_network/CMakeLists.txt

```diff
@@ -71,8 +76,10 @@ install(
 # Backend libraries
 if(NOT DEFINED ANO_MGR)
     # Initialize ANO_MGR with a default if not provided
-    # Note: rivermax not added to the default list yet since it requires licensing
-    set(ANO_MGR "dpdk gpunetio" CACHE STRING "Manager type(s) list")
+    # Default: dpdk only (no DOCA dependency)
+    # Add "gpunetio" to enable GPU-accelerated networking (requires DOCA)
+    # Note: rivermax not added to the default list since it requires licensing
+    set(ANO_MGR "dpdk" CACHE STRING "Manager type(s) list")
```

Comment:
and now need to update instructions for gpunetio [in the README](https://github.com/cliffburdick/holohub/blob/ccb13d3972d868ab2d10d13b1dee67cfa5ded446/applications/adv_networking_bench/README.md) to include the following

```
--build-args="--target gpunetio-deps" --configure-args="-D ANO_MGR:STRING=gpunetio"
```